status/discover.go

```diff
@@ -0,0 +1,76 @@
+package status
+
+import (
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/kubernetes"
+	"io/ioutil"
+)
```

Comment:
kiali has a pattern to our imports. We put standard imports first, then blank line, then third-party imports, then blank line, then kiali imports. Each section in alphabetical order (this sorting is done automatically by gofmt, which is why we break the groups up with newlines). So this import section should be:

```
import (
	"io/ioutil"

	"github.com/kiali/kiali/config"
	"github.com/kiali/kiali/kubernetes"
)
```